app/code/Magento/Fedex/Model/Source/Generic.php

```diff
@@ -36,9 +43,12 @@ public function toOptionArray()
     {
         $configData = $this->_shippingFedex->getCode($this->_code);
         $arr = [];
-        foreach ($configData as $code => $title) {
-            $arr[] = ['value' => $code, 'label' => $title];
+        if ($configData) {
+            foreach ($configData as $code => $title) {
+                $arr[] = ['value' => $code, 'label' => $title];
+            }
```

Comment:
Use `array_map` instead.

Reply:
@lbajsarowicz Thanks for the feedback, I have updated the `array_map` instead of `foreach`.

Reply:
@lbajsarowicz Thanks for the valid feedback, I have updated `array_map` instead of `foreach`. can you please review and add the labels.